mintpy/generate_mask.py

```diff
@@ -69,6 +72,8 @@ def create_parser():
                         help='maximum value for selected pixels')
     parser.add_argument('-p','--mp','--minpixels', dest='minpixels', type=int,
                         help='minimum cluster size in pixels, to remove small pixel clusters.')
+    parser.add_argument('--vstd', action='store_true',
+                        help='mask according to the formula: |velocity|>2*velocityStd')
```

Comment:
Could you add an extra option to specify the cutoff number, e.g. `--vstd-num`, with a default value of 2?

Reply:
Sure!